### DISCUSSION TITLE
Contact information that should be in the `README` ?

### DISCUSSION THREAD

**Discussion by gbraad** [crc-org/crc#4506] [General]
Currently, in our `README` we mention to reach out on Freenode, but this is probably not the right place anymore. What should we suggest ionstead?

https://github.com/crc-org/crc/blob/b1568769482937659851c9e05763c3406d7117e4/README.md?plain=1#L48

**Comment by rohanKanojia** [comment]
Related to https://github.com/crc-org/crc/pull/4495#discussion_r1872640376

**Comment by praveenkumar** [comment]
I think it should be `podman-desktop` on kubernetes slack which is not restricted to RH.
